src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -489,11 +507,49 @@ export default {
 						});
 					}
 
+					const teamPublicId = route?.query?.team ?? '';
+					const challengeHeaderExpData = client.readFragment({
+						id: `Experiment:${CHALLENGE_HEADER_EXP}`,
+						fragment: experimentVersionFragment,
+					}) ?? {};
+					const activeChallengeHeaderExp = challengeHeaderExpData?.version === 'b';
+
 					return Promise.all([
 						client.query({ query: experimentAssignmentQuery, variables: { id: SHARE_LANGUAGE_EXP } }),
 						client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
 						client.query({ query: experimentAssignmentQuery, variables: { id: EDUCATION_PLACEMENT_EXP } }),
+						teamPublicId,
+						activeChallengeHeaderExp,
 					]);
+				}).then(response => {
+					const teamPublicId = response[3];
+					const activeChallengeHeaderExp = response[4];
+
+					let teamDataPromise = Promise.resolve();
+					if (teamPublicId && activeChallengeHeaderExp) {
+						teamDataPromise = client.query({ query: TeamInfoFromId, variables: { team_public_id: teamPublicId } }); // eslint-disable-line max-len
```

Comment:
Can't this query can be ran in the `Promise.all` on line 517? I don't see anything that this `.then` needs from that `Promise.all`.